substrate/frame/election-provider-multi-block/src/signed/mod.rs

```diff
@@ -163,39 +164,54 @@ impl<T: Config> SolutionDataProvider for Pallet<T> {
 						Precision::BestEffort,
 					);
 					debug_assert!(_res.is_ok());
+				} else {
+					// No leader to reward; nothing to do.
+					sublog!(
+						warn,
+						"signed",
+						"Tried to report Queued but no leader was present for round {}",
+						current_round
```

Comment:
Both `DataUnavailableInfo::Page` and `DataUnavailableInfo::Score` are equally defensive, and should never happen. Can we handle them in the same way, in a more concise way?